rules/const.bzl

```diff
@@ -173,3 +177,107 @@ def error_redact(error, lc_state, redact):
     if redact == CONST.SHUTDOWN.REDACT.ALL or redact not in _REDACTION_MAP:
         return CONST.BFV.UNKNOWN
     return error & _REDACTION_MAP.get(redact)
+
+# Must match with top_earlgrey_alert_id in hw/top_earlgrey/sw/autogen/top_earlgrey.h
+EARLGREY_ALERTS = [
```

Comment:
nit: given that we convert these into `set`s, shouldn't these be defined as such?

Reply:
They are converted to sets for input validation, but we need to keep the order of the alerts in the OTP somewhere. The final line that generates the OTP entries uses the original list.

Skylib docs say that the sets are implemented as [keys in a dict](https://github.com/bazelbuild/bazel-skylib/blob/main/docs/new_sets_doc.md), and Bazel then guarantees that a dict will [maintain insertion order](https://bazel.build/rules/lib/core/dict), but it seems like a bad idea to rely on skylib's implementation details for this especially since it would fail silently if the order wasn't preserved. I would rather just guarantee the correct ordering and do some extra computation.

Reply:
SGTM, can you add a comment to emphasize that the type is actually important for correctness here?